### PR TITLE
Use xz instead of bzip2

### PR DESCRIPTION
**Pull request by brianhlin**
https://lwn.net/Articles/762264/

### CODE REVIEW

**Comment by brianhlin** [comment]
Tested locally, merging

File: s3-backup.sh [hunk 1]
```diff
@@ -43,7 +43,7 @@ mc_backup () {
     tmpdir=$(mktemp -d)
     chmod 700 $tmpdir
     BACKUP=$tmpdir/backup-$(date +%Y%m%d-%H%M).tar
```
Comment (jthiltges) [inline s3-backup.sh:45]: Should the filename pattern now be `backup-DATE.tar.xz.enc` or even just `backup-DATE.enc`?

Comment (brianhlin) [reply s3-backup.sh:45]: I was on the fence about changing it but I guess it does make more sense to add the compression extension. I'll submit another PR

Comment (jthiltges) [reply s3-backup.sh:45]: For sanity, seems to make sense having it all (tar.xz.enc) or nothing (enc). Probably best with the extension, then tar's `--auto-compress` will do the right thing for any changes down the road.

**Pull request merged**